spec/rspec/core/bisect/runner_spec.rb

```diff
@@ -14,13 +14,14 @@ module RSpec::Core
           [], %w[ spec/failure_spec.rb[1:1] spec/failure_spec.rb[1:2] ]
         )
 
+        # TODO: expectations here are weird
```

Comment:
"Weird" is bad phrasing on my part. "Repetitive" might be a better description. I'll see if I can dry them out a bit.